### PR TITLE
Blog: introducing hierarchical namespaces

### PR DESCRIPTION
**Pull request by adrianludwin**
The draft of this blog post was reviewed by two other leads of wg-multitenancy (@tashimi and @rjbez17) as well as by @mrbobbytables .

### CODE REVIEW

**Comment by adrianludwin** [comment]
I've made a few minor changes since this was staged to netifly; I don't know if those will be picked up automatically or if someone needs to restage.

One thing I'm not sure about is footnotes. I added two footnotes with manual links; PTAL if there's a better way to do that, or if you prefer to have no footnotes at all.

File: content/en/blog/_posts/2020-08-17-introducing-hierarchical-namespaces.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  Kind:  ClusterRole
+  Name:  admin
+Subjects: ...
+```
+
+Finally, HNC adds labels to these namespaces with useful information about the
+hierarchy which you can use to apply other policies. For example, you can create
+the following Network Policy:
+
+```
```
Comment (sftim) [inline content/en/blog/_posts/2020-08-17-introducing-hierarchical-namespaces.md:144]: This one you can mark as YAML if you want to.

File: content/en/blog/_posts/2020-08-17-introducing-hierarchical-namespaces.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+can only be applied by HNC, and is guaranteed to reflect the latest hierarchy.
+
+You can learn all about the features of HNC from the [user
+guide](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/hnc/docs/user-guide).
+
+# Next steps and getting involved
+
+If you think that hierarchical namespaces can work for your organization, [HNC
+v0.5.1 is available on
+Github](https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.5.1).
```
Comment (sftim) [inline content/en/blog/_posts/2020-08-17-introducing-hierarchical-namespaces.md:170]: nit:
```suggestion
GitHub](https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.5.1).
```

Comment (adrianludwin) [reply content/en/blog/_posts/2020-08-17-introducing-hierarchical-namespaces.md]: Done

File: content/en/blog/_posts/2020-08-17-introducing-hierarchical-namespaces.md (hunk 1 above)
Comment (adrianludwin) [reply content/en/blog/_posts/2020-08-17-introducing-hierarchical-namespaces.md]: Done

**Comment by adrianludwin** [comment]
> On this website it's more common to write ResourceQuota, ServiceAccount and NetworkPolicy. Up to you if you want to revise.

Done - wasn't sure what to do with pluralizations but at least some places seem to say "NetworkPolicies." lmk if those should be lower-cased regular words instead. Thanks!

**Comment by adrianludwin** [comment]
@sftim , I wouldn't mind publishing this earlier than Aug 17 if we're ready. LMK if you want to publish earlier and I'll change the dates, filenames, etc.

**Comment by mrbobbytables** [comment]
lgtm from a tech review perspective 👍 
With another sign off we can update the dates and publish.

**Comment by onlydole** [comment]
@adrianludwin and @mrbobbytables - this looks good to me! Once the dates are updated, @ me and we should be good in my opinion 👍🏼 

/lgtm

**Comment by adrianludwin** [comment]
Thanks @onlydole! I've changed the date to Fri Aug 14 but I think that it's fine to publish at any time now.

Do you need me to squash my commits or will that happen automatically?

**Comment by onlydole** [comment]
Thank you, @adrianludwin! Bob added the `tide/merge-method-squash` label, which will take care of squashing down this PR, so you are good on that front. 

I've pinged our other reviewers for a final approval, and you should be all set. Thank you again, @adrianludwin!

**Comment by adrianludwin** [comment]
Great, thanks!

**Comment by mrbobbytables** [comment]
/approve
/hold
we can release the hold tomorrow 👍

**Pull request merged**